.travis.yml

```diff
@@ -32,11 +32,11 @@ env:
     - SPRING_JPA_SHOW_SQL=false
 
     # if JHIPSTER_LIB_BRANCH value is release, use the release from Maven
-    - JHIPSTER_LIB_REPO=https://github.com/jhipster/jhipster.git
-    - JHIPSTER_LIB_BRANCH=release
+    - JHIPSTER_LIB_REPO=https://github.com/erikkemperman/jhipster.git
```

Comment:
I assume this was an accident :smiling_imp:

Reply:
Yes and no — it should not be merged, obviously, but I don’t know how else to make Travis pass on mutually dependent changes across server-lib and generator? See also my note above.

If approved these should both be merged (minus the “temp” commits) and pushed simultaneously, then it should work (this is what I did with my branches too).

If there is a better way to do this I’d love to learn about it!

Reply:
PS I could do the merge myself, if approved by you guys, I know this “temp” business is annoying!

Reply:
Shouldn't the change for server lib be done first? so that once we merge that we can merge it here, of course the generator tests on server lib might fail but if you have tested it locally I guess it should be good. But ya in general as long as you take care not to have unwanted changes ending up in master its fine.

Reply:
Indeed, I like the reassurance of both builds passing... Merging the server-lib PR first (despite red Travis) and then the generator later would also work, but requires intervention by Julien to publish the snapshot.

I understand the arguments against it, and this kind of situation is relatively infrequent I guess, but the mono-repo idea would have helped here ;-)